eduNEXT/edunext-platform
# And/backport mfe config api

## Description
This is a backport of multiple changes which add and modify the MFE config API

1. https://github.com/openedx/edx-platform/pull/30473
2. https://github.com/openedx/edx-platform/pull/30830

## Testing instructions

1. Add this to your tenant settings

```
    "ENABLE_MFE_CONFIG_API": true,
    "MFE_CONFIG": {
        "LMS_BASE_URL": "https://edunext.stage.nelp.gov.sa"
    },
    "MFE_CONFIG_OVERRIDES": {
        "learning": {
            "is_new": "yes"
        }
    },
```
2. Go to /api/mfe_config/v1
3. to see the app configuration add the queryparam /api/mfe_config/v1?mfe=learning

You can test it in stage with this branch https://github.com/nelc/manifest-sagar/tree/and/backport_mfe_config_api
